.github/workflows/ci.yaml

```diff
@@ -0,0 +1,36 @@
+name: CI
+
+on: [ push, pull_request ]
+
+jobs:
+  run:
+    runs-on: ${{ matrix.operating-system }}
+    strategy:
+      matrix:
+        operating-system: [ ubuntu-latest ]
+        php-versions: [ '7.2', '7.3', '7.4' ]
```

Comment:
I think we should test starting from PHP 8.0, this is what Infection itself is currently being tested on

related to https://github.com/infection/infection/pull/1765